QUARKUS-728.md

```diff
@@ -21,7 +24,7 @@ Test development will focus on
 
 ### Impact on testsuites and testing automation:
  - Ensure this coverage works in JVM mode
- - NATIVE mode is not required in Java 17 
+ - Ensure this coverage works in native mode (Madrel images) 
```

Comment:
Mandrel image based on Java 17 should be covered in extended platform testing, it's not the primary Mandrel variant.
So just subset of testing coverage will be executed with Mandrel image based on Java 17.